example.js

```diff
@@ -0,0 +1,25 @@
+
+var Stagger = require('./lib/stagger');
+
+var stagger = Stagger.getDefault({
+  "pid": "1234",
+  "cmd": "./hello"
```

Comment:
getDefault is a singleton so it can't be configurable, it's used in various places inside of our code. The library should find out the current pid and command from the process by default.